nuxt.config.ts

```diff
@@ -40,5 +41,70 @@ export default /*<NuxtConfig>*/ {
     server: 'https://ackee.nuxtjs.com',
     domainId: 'ab823e69-2425-4a16-85c8-bd9b42d11e1e',
     detailed: true
+  },
+  feed() {
+    const feedFormats = {
+      rss: { type: 'rss2', file: 'rss.xml' },
+      json: { type: 'json1', file: 'feed.json' },
+    }
+    const { $content } = require('@nuxt/content')
+
+    const createFeedItems = async feed => {
+      feed.options = {
+        title: 'Nuxt Modules',
+        link: 'https://modules.nuxtjs.org/',
+        description: 'Discover our list of modules to supercharge your Nuxt project. Created by the Nuxt team and community.',
+      }
+
+      const modules = await $content()
+        .sortBy('createdAt', 'desc')
+        .limit(10)
+        .fetch()
+
+      for (const { name, website, createdAt, description, icon, category, type, maintainers } of modules) {
```

Comment:
I asume `createdAt` is based on `fs.stat` for module meta, which should be fine. /cc @Atinux @benjamincanac

Reply:
Not sure this answers your question, but `createdAt` is supplied by `$content`. From what I see this project uses the default implementation.

Reply:
It should be good normally, it is based from `fs.stat`, I may be afraid that it won't respect the filesystem date when deploying on Vercel (https://github.com/nuxt/content/blob/dev/packages/content/lib/database.js#L243)

Reply:
Hmm that's definitely an issue with vercel then :( Maybe we can utilize https://github.com/MestreLion/git-tools/blob/master/git-restore-mtime

Reply:
Just curious, is this a general problem with the content module + Vercel? Or is it just an issue for this project?

Reply:
I think this should be an issue in most providers since they actually don't really clone the repo but download the tar.

Reply:
Looks like it is the same date: https://modules-3ergmki3y.vercel.app/rss.xml